src/include/sof/lib/dai.h

```diff
@@ -86,6 +86,7 @@ struct dai_ops {
 	int (*remove)(struct dai *dai);
 	uint32_t (*get_init_delay_ms)(struct dai *dai);
 	int (*get_fifo_depth)(struct dai *dai, int direction);
+	void (*copy)(struct dai *dai);
```

Comment:
A bit of documentation would be nice. In the context of the dmic problem this is easily understandable, but if you zoom out a bit, a copy method with no arguments looks very odd. Copy what and where?